### PR TITLE
dsa

### PR DESCRIPTION
**Pull request by userNotAvailableE**
@cyrusjetson check this @cyrusjetson

### CODE REVIEW

File: Datastructures.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import java.util.*;
+
+public class Datastructures {
+    public static void main(String[] args) {
+//        // arrays
+//        String [] colors = new String[5];
+//        colors[0] = " hai";
+//        System.out.println(Arrays.toString(colors));
+//        System.out.println(colors); // address
+//
```
Comment (cyrusjetson) [inline Datastructures.java:12]: nice but bettr use comments

Comment (userNotAvailableE) [reply Datastructures.java:12]: sure

File: Datastructures.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+//
+//        /// removing
+//        student.remove(2);
+//        System.out.println(student.get(2)); // returns null
+//        System.out.println(student);
+//        student.replace(1,"hkk");
+//        System.out.println(student);
+
+// hash
```
Comment (cyrusjetson) [inline Datastructures.java:131]: i changed here

File: Datastructures.java [hunk 3]
```diff
@@ -128,9 +128,7 @@ public static void main(String[] args) {
 //        student.replace(1,"hkk");
 //        System.out.println(student);
 
-
-        // Hash code ,,, hash function
```
Comment (userNotAvailableE) [inline Datastructures.java]: kk

Comment (cyrusjetson) [reply Datastructures.java]: ..

**Pull request merged**